scripts/jobs/top_10_most_expensive_queries.sql

```diff
@@ -0,0 +1,34 @@
+-- Top 10 most expensive queries, last 7 days.
```

Comment:
Seems like this file shouldn't exist because you've placed one in the sql/ subdirectory